src/main/java/carpet/script/utils/ShapeDispatcher.java

```diff
@@ -121,7 +133,7 @@ public static void sendShape(Collection<ServerPlayer> players, List<ShapeWithCon
         List<ServerPlayer> alternativePlayers = new ArrayList<>();
         for (ServerPlayer player : players)
         {
-            if (ServerNetworkHandler.isValidCarpetPlayer(player))
+            if (ServerNetworkHandler.isValidCarpetPlayer(player)&&!CarpetSettings.superSecretSetting)//for debug
```

Comment:
Correct me if I'm wrong, but I thought that we filter out carpet client players because non-carpet clients can't receive the shapes correctly?

Reply:
isValidCarpetPlayer is already checking for super secret settings - that is not needed

Reply:
fixed